lib/registration.pm

```diff
@@ -791,6 +791,12 @@ sub fill_in_reg_server {
         $regcode = $prdcode if ($prdcode);
     }
 
+    # rmt slp discovery, the smt server address will auto filled
```

Comment:
Add comments here that is to avoid the impact to other openqa test, we will run SLP test on local environment.

Reply:
done.